app/controllers/visitors_controller.rb

```diff
@@ -6,13 +6,15 @@ def index
     get_events
     @tree = ancestry_options(Area.unscoped.arrange(:order => 'title')) {|i| "#{'-' * i.depth} #{i.title}" }
     @holders = get_holders_by_area(params[:area])
+    get_holder
```

Comment:
@raul-fuentes ¿Why do we need this code here? Maybe i am missing something 🤔 

In this action, holders are chosen from search form selector, so they always exist ...

![captura de pantalla 2018-01-10 a las 10 03 08](https://user-images.githubusercontent.com/15726/34764043-7eb1f8aa-f5ed-11e7-863a-20e068a76f5a.png)

I think we need redirection only at visitors#agenda action when given param holder does not exist in database